plugins/amdgpu/amdgpu_plugin.c

```diff
@@ -64,6 +67,8 @@ bool plugin_added_to_inventory = false;
 
 bool plugin_disabled = false;
 
+bool parallel_disabled = false;
```

Comment:
Could you please add a comment to clarify why/when parallel restore is disabled?

Reply:
OK, I will add it in the next version.